lasagne/layers/recurrent.py

```diff
@@ -106,8 +106,13 @@ class CustomRecurrentLayer(MergeLayer):
     input_to_hidden : :class:`lasagne.layers.Layer`
         :class:`lasagne.layers.Layer` instance which connects input to the
         hidden state (:math:`f_i`).  This layer may be connected to a chain of
-        layers, which must end in a :class:`lasagne.layers.InputLayer` with the
-        same input shape as `incoming`.
+        layers, which must end in a :class:`lasagne.layers.InputLayer`.
+        ``input_to_hidden.input_shape[1:]`` should always be the same as
```

Comment:
Maybe just start this off with: `... InputLayer with the same input shape as incoming, except for the first dimension: This must either be None, or ... (when precompute_input=True), or ... (when precompute_input=False).`